runner/scheduler.py

```diff
@@ -54,11 +54,10 @@ class TestRunner:
     in_progress_results: dict[str, TestResult] = field(default_factory=dict)
     traveller: Optional[time_machine.travel] = None
     wait_list: dict[str, dict[str, int | datetime]] = field(default_factory=dict)
-    current_token_count: int = 0
-    avg_tokens_per_second: Optional[float] = None
+    agent_token_count: int = 0
```

Comment:
So `agent_token_count` is only for tokens generated by the agent and is only used to calculate the tokens per second and speed of the agent?

Reply:
That's right. And we haven't used that metric so far, although it seems useful. We do use the _tests per hour_ metric, which I guess it's an equivalent. We could also maybe use `avg_tokens_per_second` for estimations, but I'm not sure which.